sdk/search/azure-search-documents/src/test/resources/session-records/closingTwiceDoesNotThrow.json

```diff
@@ -0,0 +1,4 @@
+{
+  "networkCallRecords" : [ ],
```

Comment:
At this time, yes. This is an area we need to improve on for test recording. Without this empty data playback file the test will fail because its recorded data will be null.